crates/bevy_ecs/src/system/query.rs

```diff
@@ -1,16 +1,15 @@
 use crate::{
-    component::{Component, Tick},
+    component::Tick,
     entity::Entity,
     query::{
-        BatchingStrategy, QueryCombinationIter, QueryComponentError, QueryEntityError, QueryIter,
-        QueryManyIter, QueryParIter, QuerySingleError, QueryState, ROQueryItem, ReadOnlyWorldQuery,
-        WorldQuery,
+        BatchingStrategy, QueryCombinationIter, QueryEntityError, QueryIter, QueryManyIter,
+        QueryParIter, QuerySingleError, QueryState, ROQueryItem, ReadOnlyWorldQuery, WorldQuery,
     },
-    world::{unsafe_world_cell::UnsafeWorldCell, Mut},
+    world::unsafe_world_cell::UnsafeWorldCell,
 };
-use std::{any::TypeId, borrow::Borrow};
+use std::borrow::Borrow;
 
-/// [System parameter] that provides selective access to the [`Component`] data stored in a [`World`].
```

Comment:
Revert this and keep the backticks for formatting purposes. The ref-link can use the backticks too.

Reply:
My bad on the documentation! Fixed up.